toolkits/global/packages/global-forms/README.md

```diff
@@ -69,68 +85,125 @@ This component supports a wide range of standard form field attributes. For exam
 }
 ```
 
-#### Hints
+### Fieldsets
+
+Use a fieldset to group fields together under its `fields` property. Use a `legend` property to label the fieldset itself.
+
+![Panel](https://user-images.githubusercontent.com/15365576/209008283-cf253e6c-2ef8-4af1-8751-8a04da67c317.png)
 
-The `hint` property adds hint text under the main label text but _inside_ the `<label>`. This means it is automatically available to screen reader software.
+There are 2 ways to add fieldsets to your form:
+1. Create a field with the template value of `globalFormFieldset` and an array of fields using its `fields` property.
+2. Create an array of fieldsets and loop over them, assigning the globalFormFieldset partial.
+
+#### (1) The fieldset field
 
 ```json
-"hint": "This will be the email address you used when registering"
+{
+    "template": "globalFormFieldset",
+    "legend": "<h1>Login</h1>",
+    "fields": [
+        ...
+    ]
 ```
+    
+#### (2) A fieldset array
+In this case, each fieldset is an object:
 
-#### Required
+```json
+"myFieldsets": [
+    {
+        "legend": "<h2>One fieldset</h2>",
+        "fields": [
+            ...
+        ]
+    },
+    {
+        "legend": "<h2>Another, different fieldset</h2>",
+        "fields": [
+            ...
+        ]
+    }    
+]
+```
 
-Fields are required by default. Avoid asking for information you don't really need. To explicitly mark fields as required, `requiredSuffix` will append text—such as an asterisk—to the label. This property permits HTML, meaning you can hide the asterisk character from screen readers.
+And the accompanying template would iterate over the fieldsets, applying the `globalFormFieldset` partial:
```

Comment:
Not clear what 'iterate' means in this context?

Reply:
@Heydon can you clarify here?

Reply:
@alexhallspringer @foxintherain Sorry, writing for developers! It's a common term for traversing data using a loop. Could forego that with:

_"And the accompanying template would apply the `globalFormFieldset` partial to each fieldset."_

Reply:
```suggestion
And the accompanying template would apply the `globalFormFieldset` partial to each fieldset:
```